client/webpack.config.js

```diff
@@ -57,7 +57,11 @@ module.exports = function exports(env, argv) {
       'workflow-status',
       'bulk-actions',
     ],
-    'images': ['image-chooser', 'image-chooser-telepath'],
+    'images': [
+      'image-chooser',
+      'image-chooser-modal',
+      'image-chooser-telepath',
+    ],
     'documents': ['document-chooser', 'document-chooser-telepath'],
```

Comment:
Don’t we need to make the same change as above for the document chooser?

Reply:
I think so, but funny thing is the document chooser modal file was already inside the entrypoints file weirdly enough